gloo/docs/enterprise/installation/install_kubernetes.md

```diff
@@ -69,7 +70,7 @@ received either as part of your subscription or as part of a trial request from
 {{% /notice %}}
 
 {{% notice info %}}
-You can install Gloo to an existing namespace by providing the `-n` option, e.g. `glooctl install gateway -n my-namespace`.
+You can install Gloo to an existing namespace by providing the `--namespace` option, e.g. `glooctl install gateway --namespace my-namespace`.
```

Comment:
Curious: why are you expanding this?

Reply:
I know for non-expert users it's helpful to see the expanded option name so they know what it does. Lots of `-xyz` obscure options can be overwhelming and confusing to new users. And since Kubectl (and glooctl) have these nice option names we may as well use in our documents.